Doc/library/stdtypes.rst

```diff
@@ -4309,6 +4315,14 @@ pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
            dict(mapping, **kwarg)
            dict(iterable, **kwarg)
 
+   Dictionaries can be created by several means:
+
+   * Use a comma-separated list of ``key: value`` pairs within braces:
+     ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098: 'jack', 4127: 'sjoerd'}``
+   * Use a dict comprehension: ``{}``, ``{x: x ** 2 for x in range(10)}``
```

Comment:
Use -> Using

Also, could we make `{}` part of the first item? It's the most common way to build a dict so it should be more pre-eminent, it's also not really a dictionary comprehension.

Reply:
I would not change Use to Using as Use is the imperative form and highly preferred. See also https://github.com/python/cpython/pull/995/files#r120774265